insights/specs/sos_archive.py

```diff
@@ -202,11 +202,6 @@ class SosSpecs(Specs):
     rabbitmq_report_of_containers = glob_file("sos_commands/rabbitmq/docker_exec_-t_rabbitmq-bundle-docker-*_rabbitmqctl_report")
     rabbitmq_startup_err = simple_file("/var/log/rabbitmq/startup_err")
     rhn_charsets = first_file(["sos_commands/satellite/rhn-charsets", "sos_commands/rhn/rhn-charsets"])
-    rhn_hibernate_conf = first_file(["/usr/share/rhn/config-defaults/rhn_hibernate.conf", "/config-defaults/rhn_hibernate.conf"])
-    rhn_search_daemon_log = first_file([
-        "/var/log/rhn/search/rhn_search_daemon.log",
-        "/rhn-logs/rhn/search/rhn_search_daemon.log"
-    ])
```

Comment:
@Rohan21Lobo please do not remove these specs as we may still receive sosreports that include these files.

Reply:
Added them back.